fastlane/config/freemium/metadata/de-DE/description.txt

```diff
@@ -33,7 +33,7 @@ CRYPTOMATOR COMMUNITY
 
 Tritt der Cryptomator Community bei und tausche dich mit Cryptomator-Nutzern aus: https://community.cryptomator.org
 
-• Folge uns auf Twitter @Cryptomator
+• Folge uns auf Mastodon <a href="https://mastodon.online/@cryptomator">@cryptomator@mastodon.online</a>
```

Comment:
Unfortunately, clickable links are not possible in App Store descriptions. (Same for the other 3 instances.)